demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -164,17 +168,16 @@ def _search_packs_by_items_names(items_names, items_list, exclude_ignored_depend
         if not isinstance(items_names, list):
             items_names = [items_names]
 
-        content_items = list(
-            filter(lambda s: list(s.values())[0].get('name', '') in items_names and 'pack' in list(s.values())[0],
-                   items_list))
-
-        if content_items:
-            pack_names = list(map(lambda s: next(iter(s.values()))['pack'], content_items))
-            if not exclude_ignored_dependencies:
-                return set(pack_names)
-            return {p for p in pack_names if p not in constants.IGNORED_DEPENDENCY_CALCULATION}
+        pack_names = set()
+        for item in items_list:
+            item_details = list(item.values())[0]
+            if item_details.get('name', '') in items_names and 'pack' in item_details and \
+                    LooseVersion(item_details.get('toversion', '99.99.99')) >= MINIMUM_DEPENDENCY_VERSION:
+                pack_names.add(item_details.get('pack'))
 
-        return None
```

Comment:
Just update the return value in doc string in such case, may be confusing later.